frontend/pages/profile/connections.js

```diff
@@ -0,0 +1,117 @@
+import { useState, useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import ProfileNavigation from '@/components/ProfileNavigation';
+import withAuth from '@/components/withAuth';
+
+import ProfileLayout from '@/components/ProfileLayout';
+
+const MyConnections = () => {
+  const { user } = useSelector((state) => state.auth);
+  const [connections, setConnections] = useState({
+    all: [],
+    accepted: [],
+    pending: [],
+    rejected: [],
+  });
+  const [activeTab, setActiveTab] = useState('all');
+
+  useEffect(() => {
+    const loadConnections = async () => {
+      if (user) {
+        const receivedRequests = user.connection_requests_received || [];
+        const sentRequests = user.connection_requests_sent || [];
+
+        const allConnections = [...receivedRequests, ...sentRequests];
+
+        const acceptedConnections = allConnections.filter(
+          (connection) => connection.connection_status === 'Accepted'
+        );
+        const pendingConnections = allConnections.filter(
+          (connection) => connection.connection_status === 'Pending'
+        );
+        const rejectedConnections = allConnections.filter(
+          (connection) => connection.connection_status === 'Rejected'
+        );
+
+        setConnections({
+          all: allConnections,
+          accepted: acceptedConnections,
+          pending: pendingConnections,
+          rejected: rejectedConnections,
+        });
+      }
+    };
+
+    loadConnections();
+  }, [user]);
+
+  console.log(user);
+  const renderConnections = () => {
+    const filteredConnections = connections[activeTab];
+
+    return filteredConnections.map((connection) => (
+      <div
+        key={connection.id}
+        className="flex items-center justify-between p-4 border-b hover:bg-gray-50"
+      >
+        <div className="flex items-center space-x-4">
+          <div className="w-12 h-12 rounded-full bg-gray-200">
+            {/* Profile Image */}
+          </div>
+          <div>
+            <h3 className="font-semibold">
+              {connection.documentId || connection.full_name}
```

Comment:
Here as per design we need to show the connection name. But with current model of stakeholder connections we are only getting documentId we should get full_name from user as well as the image